src/App.tsx

```
@@ -1,10 +1,106 @@
+// ❌ More violations
+const usr = "john_doe"; // ❌ Abbreviation
+const cfg = { timeout: 5000 }; // ❌ Abbreviation + magic number + missing units
+const mgr = "manager"; // ❌ Abbreviation
+const temp = 98.6; // ❌ Missing units (should be tempInFahrenheit)
+const delay = 3000; // ❌ Missing units (should be delayInMs)
+
+console.log(A, B); // ❌ Console.log in production
+console.debug("Debug info"); // ❌ Console debug
+
+// ❌ Function with boolean parameter
+function process(data: any, shouldValidate: boolean) {
```

ESLint

[error] 'process' is defined but never used.

(@typescript-eslint/no-unused-vars)